intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/plugin/PluginVendors.kt

```diff
@@ -0,0 +1,33 @@
+package com.jetbrains.plugin.structure.intellij.plugin
+
+const val CORE_PLUGIN_ID = "com.intellij"
```

Comment:
Why not inside `PluginVendors` object?

Reply:
Fixed. This will be coordinated with the #952.